packages/mongo/package.js

```diff
@@ -9,7 +9,7 @@
 
 Package.describe({
   summary: "Adaptor for using MongoDB and Minimongo over DDP",
-  version: '1.5.0-beta17.23'
+  version: '1.5.1-beta17.23'
```

Comment:
Let's revert this change back to `1.5.0-beta17.23`, since 1.5.0 hasn't been released yet, and won't be released until Meteor 1.7 is released. Generally, when we bump the minor or major version of a package as part of a Meteor release, any changes to that package can be lumped into that one version.